### DISCUSSION TITLE
现在微软官方下场做了Dev-Home，，实现了winget的图形化，您这边还继续做 WinGet-Store吗？

### DISCUSSION THREAD

**Discussion by KyoSakuyo** [wherewhere/WFun-UWP#16] [Q&A]
估计不做了吧，哈哈。

**Comment by wherewhere** [comment]
什么鬼，这里是WFun

**Reply by KyoSakuyo** [reply]
![图片](https://github.com/wherewhere/WFun-UWP/assets/59904064/ee5efedc-098e-4665-9734-9dd32bfa3c26)
欸，那你是不是项目的设置里面是不是有什么地方没做好啊，我从WinGet-Store的Issues里面的入口走，直接进了这里欸！

**Reply by wherewhere** [reply]
我复制粘贴的。。。
